tests/ttnn/unit_tests/gtests/tensor/test_create_tensor.cpp

```diff
@@ -65,6 +65,9 @@ INSTANTIATE_TEST_SUITE_P(
     CreateTensorTestWithShape,
     CreateTensorTest,
     ::testing::Values(
-        CreateTensorParams{.shape=ttnn::SimpleShape({1, 1, 32, 32})}
+        CreateTensorParams{.shape=ttnn::SimpleShape({1, 1, 32, 32})},
+        CreateTensorParams{.shape=ttnn::SimpleShape({2, 1, 32, 32})},
+        CreateTensorParams{.shape=ttnn::SimpleShape({0, 0, 0, 0})},
+        CreateTensorParams{.shape=ttnn::SimpleShape({0, 1, 32, 32})}
```

Comment:
will it work for these 2 cases too or not yet?
```suggestion
        CreateTensorParams{.shape=ttnn::SimpleShape({0})},
        CreateTensorParams{.shape=ttnn::SimpleShape({})}
```

Reply:
Added those shapes and made them work